src/Request.js

```diff
@@ -8,6 +8,8 @@ const { tokenize, parseActionPayload } = require('./utils');
 const { disambiguationQuickReply, quickReplyAction } = require('./utils/quickReplies');
 const { getSetState } = require('./utils/getUpdate');
 const { vars } = require('./utils/stateVariables');
+const OrchestratorClient = require('./OrchestratorClient');
+const { media } = require('./resolvers');
```

Comment:
tohle tu chybelo, nebo to prebyva?

Reply:
'OrchestratorClient' tam chybelo kvuli typu /** @type {import('./OrchestratorClient').OrchestratorClientOptions} */. Media tam přebývalo, smazal jsem.